CHANGELOG.md

```diff
@@ -45,6 +45,11 @@
   - `-redis.password` --> `-store.chunks-cache.redis.password`
   - `-redis.timeout` --> `-store.chunks-cache.redis.timeout`
 * [CHANGE] Rename the `-store.chunk-cache-stubs` to `-store.chunks-cache.cache-stubs` to be more inline with above.
+* [CHANGE] Change prefix of flags `-dynamodb.periodic-table.*` to `-table-manager.index-table.*`. #2359
```

Comment:
Weren't these already removed by #2221 ?

Reply:
Oh I get it - these are the scaling flags.  So the description here could be made clearer by saying that.